manifest.json

```
@@ -39,6 +61,18 @@
           "is_required": true,
           "hint": "Select a user in the workspace",
           "name": "user_id"
+        },
+        "button_behavior": {
+          "type": "string",
+          "title": "Button behavior",
+          "description": "Message recipient",
+          "is_required": true,
+          "hint": "Select a user in the workspace",
```

```suggestion
          "description": "Action of the button",
          "is_required": true,
          "hint": "Decide the message sent with the button",
```

This was catching me in WFB - I'm super open to other wording 🙏